client/src/app/core/services/collectionStringModelMapper.service.ts

```diff
@@ -1,4 +1,4 @@
-import { ModelConstructor, BaseModel } from '../../shared/models/base.model';
+import { ModelConstructor, BaseModel } from '../../shared/models/base/base.model';
```

Comment:
naming is inconsistent now. Plain old objects don't need to be named `*.model.ts`, especially since they are in the `/models/` folder which indicates that they all are models.

Angular itself names `.services` and `.components` accordingly.

If we go this way to explicitly declare plain old objects using `.model.ts` and `.interface.ts` in the filename, we would need to rename all the files. 

furthermore, interface might go into their own directory and the `ModelConstructor` might also have it's own file, since a lot of components seem to depend in it now.